Extension/c_cpp_properties.schema.json

```diff
@@ -26,17 +26,22 @@
                         }
                     },
                     "cStandard": {
-                        "description": "Version of the C language standard to use for IntelliSense.",
+                        "description": "Version of the C language standard to use for IntelliSense. Note: GNU standards are only used to query the set compiler to get GNU defines and IntelliSense will emulate the equivalent C standard version.",
```

Comment:
Is description clear? please advice if changes are needed.

Reply:
Yeah, it should be "defines, and" though. Also, "query the set compiler" might be more easy to understand as "query the compilerPath`.